Formula/cdrdao.rb

```diff
@@ -21,11 +21,13 @@ class Cdrdao < Formula
   depends_on "libvorbis"
   depends_on "mad"
 
-  # first patch fixes build problems under 10.6
-  # see https://sourceforge.net/p/cdrdao/patches/23/
-  patch do
-    url "https://sourceforge.net/p/cdrdao/patches/_discuss/thread/205354b0/141e/attachment/cdrdao-mac.patch"
-    sha256 "ee1702dfd9156ebb69f5d84dcab04197e11433dd823e80923fd497812041179e"
+  on_macos do
+    # first patch fixes build problems under 10.6
+    # see https://sourceforge.net/p/cdrdao/patches/23/
```

Comment:
Do those issues still exist 10 major macOS versions later?

Reply:
Yes, I tested without the patch on my Mac:

```
In file included from ScsiIf-osx.cc:22:
./decodeSense.cc:86:3: error: use of undeclared identifier 'log_message'
  log_message(-2, "SCSI command failed:");
  ^
./decodeSense.cc:87:3: error: use of undeclared identifier 'log_message'
  log_message(-2, "  sense key 0x%x: %s.", code,
  ^
./decodeSense.cc:91:5: error: use of undeclared identifier 'log_message'
    log_message(-2, "  additional sense code: 0x%x", buf[0x0c]);
    ^
./decodeSense.cc:94:5: error: use of undeclared identifier 'log_message'
    log_message(-2, "  additional sense code qualifier: 0x%x", buf[0x0d]);
    ^
4 errors generated.
```